### PR TITLE
Fix Misc. Bug Reports from the Issues Page

### PR DESCRIPTION
**Pull request by Chaseshak**
See each individual commit for which issue it fixes.

### CODE REVIEW

**Comment by Chaseshak** [comment]
@rbrezynski Requesting your review here since you reported all the bugs. If you want to take a look at what I newly created and see if it appropriately fixes all of the referenced issues that would be great!

File: app/controllers/contacts_controller.rb [hunk 1]
```diff
@@ -18,28 +18,49 @@ def update
...
-    else
-      flash[:alert] = "Could not find team #{params[:name].capitalize}!"
-    end
+    flash[:danger] = 'Could not delete teams at this time.' unless destroy_roles
     redirect_to action: 'index'
   end
 
   private
 
+  def destroy_roles
```
Comment (rbrezynski) [inline app/controllers/contacts_controller.rb]: screen shot 2017-12-05 at 1 26 37 pm
If I click "select teams to delete" and then click cancel without deleting or selecting, I get this error

Comment (rbrezynski) [reply app/controllers/contacts_controller.rb]: If I do select a team to delete and then click cancel, it still deletes the team

Comment (Chaseshak) [reply app/controllers/contacts_controller.rb]: @rbrezynski Can you try it out again? I've fixed the errors.

**Comment by rbrezynski** [comment]
@Chaseshak All good to merge!

**Pull request merged**